config/modules_statuses.json

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
Sorry, don't delete this file. Just restore it to the original from the `dev` branch

Reply:
Okay sure